Sources/Uptech Helpers/Extensions/Foundation/Array+extensions.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Array+extensions.swift
+//  
+//
+//  Created by Sergey Kletsov on 25.03.2022.
+//
+
+import Foundation
+
+public extension Array {
+		/// Returns the element at the specified index if it is within bounds, otherwise nil.
+		subscript(safe index: Index) -> Element? {
+				indices.contains(index) ? self[index] : nil
+		}
+		
+		/// Returns the elements at the specified range if it is within bounds, otherwise empty array.
+		subscript(safe range: Range<Index>) -> [Element] {
+				guard
+						range.startIndex >= startIndex,
+						range.endIndex <= endIndex else {
+						return []
+				}
```

Comment:
2 tabs nested indents look awful on GitHub review screen. I will create a form containing poll to review and update some points from our [code style guide](https://github.com/uptechteam/ios-cookbook/blob/master/7-code-style.md).   
IMHO we should probably migrate to 4 spaces.